### ISSUE TITLE
[BUG] - syntaxWarning in constitutive models

### ISSUE THREAD

**Issue by bknight1** [underworldcode/underworld3-archive#203]
**Describe the bug**
- What is the bug?
```
../underworld3/src/underworld3/constitutive_models.py:634: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if inner_self.yield_stress_min.value is not 0:
```
- What version code?
latest dev branch, python-3.10.14
- What system are you running the code on?
M2 - arm
- Steps to reproduce?
Fresh install of updated dev branch

**Screenshots**

**Comment by bknight1** [comment]
Few 'is not' statements in the file, not sure if we want to update them all

**Comment by julesghub** [comment]
Yeah I say we update them. 
`!=` is a test about equality of value. 
`is not` is a test about the identify of the python object.

**Comment by bknight1** [comment]
Fixed in #222
